packages/platforms/react-native/ios/BugsnagReactNativePerformance.mm

```diff
@@ -82,12 +82,118 @@ @implementation BugsnagReactNativePerformance
     return hexStr;
 }
 
- RCT_EXPORT_METHOD(requestEntropyAsync:(RCTPromiseResolveBlock)resolve
-                   rejecter:(RCTPromiseRejectBlock)reject)
- {
-     NSString *hexStr = getRandomBytes();
-     resolve(hexStr);
- }
+RCT_EXPORT_METHOD(requestEntropyAsync:(RCTPromiseResolveBlock)resolve
+                   rejecter:(RCTPromiseRejectBlock)reject) {
+    NSString *hexStr = getRandomBytes();
+    resolve(hexStr);
+}
+
+RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(getNativeConstants) {
+    NSMutableDictionary *nativeDirs = [NSMutableDictionary new];
+    nativeDirs[@"CacheDir"] = NSSearchPathForDirectoriesInDomains(.cachesDirectory, .userDomainMask, true).first!;
+    nativeDirs[@"DocumentDir"] = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true).first!;
+    return nativeDirs;
+}
+
+RCT_EXPORT_METHOD(exists:(NSString *)path
```

Comment:
I would name the method `fileExistsAtPath`: instead of just `exists:`

Reply:
it's the same name as in replaced module - I wanted to have same interface